content/posts/newsletter/2020-07-01.md

```diff
@@ -0,0 +1,174 @@
+
+The XMPP client Dino is one out of three projects which were awarded with the [10th Thomas Krenn Award](https://www.thomas-krenn.com/de/unternehmen/presse/pressemeldung.thomas-krenn-award-2020.html) (German).
+
+## Video
+
+:es: Conociendo XMPP - The Comunidad Latina de Tecnologías Libres had an online introduction to XMPP for users back in April 2020 - we would like to reference this even if it is a bit late: https://www.youtube.com/watch?v=VwnzJ9V732E
+They also have a user chat with this short and handy MUC ID :) 
+comunidadlatinadetecnologiaslibres@chat.disroot.org 
+
+Daniel Gultsch and Holger Weiss had a virtual talk on their A/V implementation in Conversation and describe the technical details. eta finalized the session with a second talk by defining love by this formula: Asterisk + XMPP = <3
+https://www.youtube.com/watch?v=v_riuu4XU6Q
+
+:de: Prof. Dr. P. Löbbecke spoke in a virtual talk about "[Freie Messenger - Sichere Kommunikation](https://libre.video/videos/watch/1b646209-24e0-489b-8060-793f4513da9d)" ("Free messengers - secure communication) which also included XMPP (German).
+
+## Software releases
+
+_To everyone involved, please do not just drop links, rather try to make a short and decent 
+formulated description of what is the main outcome of your source. This will help us a lot! 
+You do not need to use a formal tone, be passionate about why this motivates you!_
+
+### Clients and applications
+
+_To everyone involved, please do not just drop links, rather try to make a short and decent 
+formulated description of what is the main outcome of your source. This will help us a lot! 
+You do not need to use a formal tone, be passionate about why this motivates you!_
+
+[Gajim 1.2.0](https://gajim.org/post/2020-06-21-gajim-1.2.0-released/) has been released. More than a year after the release of Gajim 1.1.3 it is finally time for Gajim 1.2. A year of developing new features, cleaning up old code, and bug fixing. Highlights of this release are (amongst others): improved group chat system, completely rewritten network code, and a new account creation assistant. But there is much more to discover.
+
+Profanity released its version 0.9.0 which supports Last Message Correction (XEP-0308) and fix various issues that can be found in their [blog post](https://profanity-im.github.io/blog/post/release-090/). After the new major release the bugfix releases [0.9.1](https://github.com/profanity-im/profanity/releases/tag/0.9.1), [0.9.2](https://github.com/profanity-im/profanity/releases/tag/0.9.2), [0.9.3](https://github.com/profanity-im/profanity/releases/tag/0.9.3) and [0.9.4](https://github.com/profanity-im/profanity/releases/tag/0.9.4) followed.
+
+[JSXC](https://jsxc.org) released [version 4.1.1](https://github.com/jsxc/jsxc/releases/tag/v4.1.1) which fixes several video related issues and enhances the interoperability with Conversations.
```

Comment:
Yesterday we released a first experimental version for Windows, Mac and Linux. Therefore it would be great if you could add the following:

Additionally a [first experimental desktop build](https://www.jsxc.org/blog/2020/06/25/JSXC-Desktop.html) for all platforms was released. The project is looking for early testers and people with experience with Electron and the Travis build chain.

Reply:
Thank you!